roles/kubernetes/preinstall/defaults/main.yml

```diff
@@ -8,7 +8,6 @@ epel_enabled: false
 dns_late: false
 
 common_required_pkgs:
-  - python-httplib2
```

Comment:
If python-httplib2 is to be removed from here it should be added to the per-distrib vars in roles/kubernetes/preinstall/vars for each other distrib

Reply:
👍